source/localizable/index.html.erb

```diff
@@ -55,16 +54,16 @@
           <%= I18n.t('top.slogan') %>
         </div>
         <div class="CTA text-center md:text-left">
-         <a class="typeform-share button" href="https://form.typeform.com/to/SksVjAuk" data-mode="popup" style="display:inline-block;text-decoration:none;background-color:#F9975F;color:white;cursor:pointer;font-family:Helvetica,Arial,sans-serif;font-size:20px;line-height:50px;text-align:center;margin:0;height:50px;padding:0px 33px;border-radius:25px;max-width:100%;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;font-weight:bold;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;" target="_blank"><%= I18n.t('top.button') %></a>
```

Comment:
why was the styling removed here?

Reply:
I put it on the stylesheet instead of in the html